dao-voting/src/abi.sw

```diff
@@ -0,0 +1,21 @@
+library abi;
+
+dep data_structures;
+
+use std::{contract_id::ContractId, identity::Identity};
+
+use data_structures::{Proposal, ProposalInfo};
+
+abi DaoVoting {
+    #[storage(read, write)] fn constructor(gov_token: ContractId);
```

Comment:
Can these be in the same order as defined in the contract? It makes it easier to follow when looking at what comes next in the main file